project/api/views.py

```diff
@@ -122,3 +124,23 @@ class NarrationViewSet(viewsets.ModelViewSet):
 
     queryset = Narration.objects.all()
     serializer_class = NarrationSerializer
+
+
+# https://medium.com/@mrgrantanderson/https-medium-com-serving-vector-tiles-from-django-38c705f677cf
+def mvt_tiles(request, zoom, x_cor, y_cor):
+    """
+    Custom view to serve Mapbox Vector Tiles for CachedData.
+    """
+
+    with connection.cursor() as cursor:
+        cursor.execute(
+            (
+                "SELECT ST_AsMVT(tile) FROM (SELECT id, wikidata_id, EXTRACT(year from date), "
```

Comment:
What is the resulted tile projection? It should be SRID:3857 (Web Mercator), All the data is in 4326, so I believe we need to do some transformations before serving MVT?